backend/inspirehep/records/serializers/json/jobs.py

```diff
@@ -19,3 +20,14 @@
 jobs_json_response = record_responsify(jobs_json, "application/json")
 
 jobs_json_response_search = search_responsify(jobs_json, "application/json")
+
+seminars_json_detail = JSONSerializer(wrap_schema_class_with_metadata(JobsDetailSchema))
```

Comment:
you don't need this here.

Reply:
beside the wrong variable name I think that it is neccessary to have this serialiser because I moved ```can_edit``` method to the ```JobsDetailSchema``` class